nav2_behavior_tree/src/behavior_tree_engine.cpp

```diff
@@ -32,6 +32,11 @@ BehaviorTreeEngine::BehaviorTreeEngine(
   for (const auto & p : plugin_libraries) {
     factory_.registerFromPlugin(loader.getOSName(p));
   }
+
+  // FIXME: the next two line are needed for back-compatibility with BT.CPP 3.8.x
```

Comment:
I added this specifically because you mentioned that you don't want to use the "latest version".

This is the workaround, if you want to use 4.5 (a small price to pay)

Reply:
OK - please file a separate ticket as a reminder to us both about this - we don't accept inline FIXME / TODOs as proper documentation of issues (there are some in the codebase, but those all predate my tenure)